public/data/aayush89890.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Aayush jain",
+    "bio": "Software Developer || YouTuber || Open Source Enthusiast",
+    "avatar": "https://avatars.githubusercontent.com/u/37421316?v=4",
```

Comment:
Please use the link, specified in the readme, so the Avatar can autoupdate whenever you change it :D

Reply:
@aayush89890 Use `https://github.com/aayush89890.png`

Reply:
@Cahllagerfeld Done!....Thanks for helping.